lab.R

```diff
@@ -36,11 +39,12 @@ dataf = read_csv(file.path('data', 'ames.csv'))
 #' 1. Look through the short descriptions in `?ames_raw` (or online, <https://cran.r-project.org/web/packages/AmesHousing/AmesHousing.pdf>).  Which variable reports the actual sale price? 
 #' 
 #' 
+#' it seems like value, except that it says that is a tibble so I think that isa tibble which i think means its another data table of value stuff? makes zero sense to me how this is organized
```

Comment:
Try searching for "price," either in the docs or the codebook: <http://jse.amstat.org/v19n3/decock/DataDocumentation.txt>

Reply:
I misunderstood this question, sorry